### PR TITLE
[PPP-4223] - Use of vulnerable component spring-core before 4.3.14, s…

### PR DESCRIPTION
**Pull request by ricardosilva88**
DO NOT MERGE!

@graimundo

### CODE REVIEW

**Comment by ricardosilva88** [comment]
DO NOT MERGE!

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 1]
```diff
@@ -98,7 +98,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-jdbc</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 2]
```diff
@@ -112,7 +112,7 @@
     <dependency>
       <groupId>org.springframework.security</groupId>
       <artifactId>spring-security-core</artifactId>
-      <version>${dependency.spring.security.version}</version>
+      <version>${spring-security.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 3]
```diff
@@ -125,7 +125,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 4]
```diff
@@ -138,7 +138,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-beans</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 5]
```diff
@@ -151,7 +151,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-web</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 6]
```diff
@@ -164,7 +164,7 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-context</artifactId>
-      <version>${dependency.spring.framework.version}</version>
+      <version>${spring.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml [hunk 7]
```diff
@@ -177,7 +177,7 @@
     <dependency>
       <groupId>org.springframework.security</groupId>
       <artifactId>spring-security-web</artifactId>
-      <version>${dependency.spring.security.version}</version>
+      <version>${spring-security.version}</version>
```
Comment (graimundo) [inline Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: Isn't this dep on the parent dep management?

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 1 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 2 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 3 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 4 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 5 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 6 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

File: Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml (hunk 7 above)
Comment (ricardosilva88) [reply Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml]: yes, agree

**Review by graimundo — APPROVED** [review]

**Comment by ricardosilva88** [comment]
This repository is no longer supported, so it doesn't make sense to update spring version

**Pull request closed without merging**